rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -242,6 +243,26 @@ def test_migration_10(
     optimism_addresses = [ethereum_accounts[2], ethereum_accounts[3]]
     polygon_pos_addresses = [ethereum_accounts[3]]
 
+    # insert a bad polygon etherscan name in the database. By mistake we published an error
+    # in this name and could affect users
+    with GlobalDBHandler().conn.write_ctx() as write_cursor:
+        # replace the packaged db polygon pos etherscan node for the bad one
+        # propagated in the data repo
+        write_cursor.execute('DELETE from default_rpc_nodes WHERE name="polygon pos etherscan"')
+        write_cursor.execute(
+            'INSERT OR IGNORE INTO default_rpc_nodes(name, endpoint, owned, active, weight, blockchain) '  # noqa: E501
+            'VALUES (?, ?, ?, ?, ?, ?)',
+            ('polygon etherscan', '', 0, 1, '0.25', 'POLYGON_POS'),
+        )
+        assert write_cursor.execute('SELECT COUNT(*) FROM default_rpc_nodes WHERE name="polygon etherscan"').fetchone()[0] == 1  # noqa: E501
+    with rotki.data.db.user_write() as write_cursor:  # Also add it to the user db
+        write_cursor.execute(
+            'INSERT OR IGNORE INTO rpc_nodes(name, endpoint, owned, active, weight, blockchain) '  # noqa: E501
```

Comment:
Doesn't this make the user DB have both of them?

Reply:
no because the test is for the migration and the db upgrade is not triggered there